apps/citizen_advocate/mix.exs

```diff
@@ -54,8 +55,10 @@ defmodule CitizenAdvocate.Mixfile do
      {:gettext, "~> 0.11"},
      {:cowboy, "~> 1.0"},
      {:comeonin, "~> 2.0"},
-     {:httpoison, "~> 0.9.0"},
+     {:httpoison, "~> 0.11.0"},
```

Comment:
the old version had a conflict with bamboo, i didn't actually check if everything still works 😬